alicptfts/alicptfts.py

```diff
@@ -231,6 +232,45 @@ def generate_headers(self, scan_params, time_start, time_end):
         headers.append(";".join(scan_params))
         return headers
 
+    def calculate_time_one_pass(self, max_range, max_velocity, accel):
+        '''
+        calculates the number of seconds it would take to move from 
+        one end of range to the other end of range
+        '''
+        vf = np.sqrt(accel*max_range)
+        if(vf < max_velocity):
+            max_time_one_pass = np.sqrt(max_range * 4 / accel)
```

Comment:
like above. these parameters may all be int
`max_time_one_pass = np.sqrt(max_range * 4. / accel)`

Reply:
fixed, although I believe python automatically converts ints to floats when necessary

Reply:
You are right. Python3 do it but python2 doesn't.